packages/d/diodon/package.yml

```diff
@@ -0,0 +1,28 @@
+name       : diodon
+version    : 1.13.0
+release    : 1
+source     :
+    - https://github.com/diodon-dev/diodon/archive/refs/tags/1.13.0.tar.gz : 387e78e24dfcac5a73c65ff758c66894a712f7d10754b4ffcfdc11590a9e0a10
+homepage   : https://github.com/diodon-dev/diodon
+license    : GPL-2.0-or-later
+component  : desktop
+summary    : Aiming to be the best integrated clipboard manager for the Unity desktop.
+description: |
+    Aiming to be the best integrated clipboard manager for the Unity desktop.
```

Comment:
Is this only target Unity? We don't have unity

Reply:
No, that is from their description. It is in Debian and they don't have Unity.

Reply:
Does debian describe it as such? It was potentially confusing for user. Maybe make it more generic

Reply:
Kinda they mention gnome as well.
https://packages.debian.org/bookworm/diodon

Reply:
Lets use the debian description. 
```
summary : GTK+ Clipboard manager
description : 
      blah blah blah
```

Change the metainfo too